<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListSelecterWidget</class>
 <widget class="QWidget" name="ListSelecterWidget">
  <layout class="QHBoxLayout" name="m_main_layout">
   <item>
    <layout class="QVBoxLayout" name="m_selecter_layout">
     <item>
      <widget class="QToolButton" name="m_scroll_up_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="arrowType">
        <enum>Qt::UpArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="m_selecter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_scroll_down_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="arrowType">
        <enum>Qt::DownArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="gui::widgets::StackedWidget" name="m_stacked_widget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::widgets::StackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>gui/widgets/StackedWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
